storage/src/main/java/org/commonjava/storage/pathmapped/core/PathDBOutputStream.java

```diff
@@ -111,6 +156,7 @@ public void close() throws IOException
             if ( checksumCalculator != null )
             {
                 checksum = checksumCalculator.getDigestHex();
+                logger.trace( "PathDBOutputStream: " +path+ " calculated checksum: " + checksum );
```

Comment:
a tiny thing. we can use
logger.trace( "PathDBOutputStream: {}, calculated checksum: {}", path, checksum );
It will avoid unnecessary param string concatenation when log level not matches.

Reply:
consistency is best. I'll update it.

Reply:
consistency is best. I'll update it.